Documentation/README.md

```diff
@@ -93,10 +93,14 @@ Name | Yes | Name of the project.
 The markup file contains textual information about the topic.
 
 ### Header
-Header node contains information about the IFC files relevant to this topic. Header has also a list of File nodes.
+Header node contains information about the IFC files relevant to this topic. 
```

Comment:
Should we maybe add _The_ at the beginning, so the sentence starts with `The header node contains...`.

I know it's not related to your change, but that would sound nicer for me😀